tools/provision.sh

```diff
@@ -32,6 +32,11 @@ else
     sudo dnf install -y ansible koji
 fi
 
+# workaround for bug https://bugzilla.redhat.com/show_bug.cgi?id=2162815
+if [[ "${ID}" == "fedora" || "$VERSION_ID" == "37" ]]; then
```

Comment:
You should have used `&&` instead of `||` here. The package gets now installed on any Fedora distro, which is not desired.

Reply:
Thanks for pointing this out, my mistake, fixed.